### PR TITLE
Update build.sbt

### PR DESCRIPTION
**Pull request by allisonport-db**
- Move `releaseProcess` to root project

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -411,7 +413,6 @@ lazy val standalone = (project in file("standalone"))
   .enablePlugins(GenJavadocPlugin, JavaUnidocPlugin)
   .settings(
     name := "delta-standalone-original",
-    skip in publish := true,
```
Comment (zsxwing) [inline build.sbt:414]: This should be kept. `standaloneCosmetic` is the project we want to publish

Comment (allisonport-db) [reply build.sbt:414]: Shouldn't it be in `skipReleaseSettings`?

Comment (allisonport-db) [reply build.sbt:414]: This can be updated: 
```
lazy val skipReleaseSettings = Seq(
  publishArtifact := false,
  skip in publish := true
)
```

Comment (zsxwing) [reply build.sbt:414]: You are right. Didn't know you were removing a duplicate config

**Review by zsxwing — APPROVED** [review]

**Pull request merged**